### PR TITLE
Fixed the bug in txt2src caused by strings containing {} that should not be replaced with getTree().

### PR DESCRIPTION
**Pull request by benjaminpjones**

### CODE REVIEW

**Comment by Lautzoballsoup** [comment]
The same bug exists in the interaction between ast2str and genomeGenerator, and my superficial attempt to fix it didn't work, so I don't want to incorporate a fix to the testing tool until we can make the same fix for the actual generator file.

Also, shouldn't the testing tool call genomeGenerator.getTree directly, rather than having the code copied/modified in ut1? Otherwise, it's possible for them to fall out of sync, so that a case could pass the testing tool but fail in an organism, or vice versa.

**Review by ELautzenhiser — CHANGES_REQUESTED** [review]
Could you put this in the Test folder?

**Review by ELautzenhiser — APPROVED** [review]

**Comment by Lautzoballsoup** [comment]
Still getting used to some github functionality - didn't mean to approve all changes, just the ast2str list change.

File: AST/ut1.py [hunk 1]
```diff
@@ -72,26 +73,23 @@ def txt2src(txt_str):
     source = f.read()
     tree = ast.parse(source)
 
-    txtTree = ast2txt(tree)
-    print txtTree
+    txt_tree = ast2txt(tree)
+    print(txt_tree)
```
Comment (Lautzoballsoup) [inline AST/ut1.py]: We should stick to Python 2 printing.

Comment (benjaminpjones) [reply AST/ut1.py]: Agreed!

**Comment by benjaminpjones** [comment]
Regarding why I copied and pasted from genomeGenerator.translate, instead of calling it:
 translate() does two things- translation from genome string, and translation to source.  txt2src only does the latter.  I did not want to muddle up ut1 with complications from codon translation.  However, I do agree that, for maintainability's sake, we should only have one function for translation to source.

Probably the best approach would be to split translate() into two separate functions in genomeGenerator and import from genomeGenerator into ut1.

**Comment by benjaminpjones** [comment]
> Could you put this in the Test folder?

Are you referring to pangram.py?

**Review by ELautzenhiser — APPROVED** [review]

**Pull request merged**